.github/ISSUE_TEMPLATE/general_issue.md

```diff
@@ -1,15 +0,0 @@
----
```

Comment:
There are bugs/problems/issues [!feature-requests] not related to playing games. So why do you want to kill the general issue template?

Reply:
I was thinking that there wasn't much of a difference between the General Issue and Bug Report (both are essentially bugs)

I can see how there's a difference between a game-related issue and non-game-related ones (like the site being down), but seems like those cases are fairly rare?
(or they can use a blank issue for those)

Reply:
My reasoning was making it as easy as possible for users without compromising info for us (bug hunters).
Debugging UI or core functions needs different info than debugging a game-related issue. Therefore the two options. Using the blank issue is worse, because then users aren’t reminded to post exactly the info we need and we’d have to ask (wasting time …).
So please revive the general issue and apply the changes you did to the platform section of the game related issue. Choose better titles for both if you’d like. (I just started with Game + General because it sounded ok and is in the right order, because we definitely have more game related bugs than everything else.).

Reply:
I see what you're saying, perhaps s/"General Issue"/"Interface or Site Issue" ?

Ah I didn't realize it would sort alpha, that makes sense to have it list "Game Issue" first

Reply:
c82a1d3